packages/website/src/docs/language-basics/intersections.md

```diff
@@ -11,7 +11,7 @@ Intersections describe a type that must include all the intersection's constitue
 alias Dog = Animal & Pet;
 ```
 
-An intersection is equivalent to [spreading]({%doc "models"%}#spread) both types.
+An intersection is equivalent to [spreading]({%doc "models", "#spread"%}) both types.
```

Comment:
Is this change worth it or the way it was done before is fine? (Both work right now)

Reply:
This is just a syntactic issue of how these refrences are made and they both come out the same. If so, I have no opinion, lol.

Reply:
yeah basically same result